### PR TITLE
Add resolve retry

### PR DESCRIPTION
**Pull request by huwshimi**
Add Resolve & retry button to the unit errors list.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- add a service
- add a bunch of units to the service
- place the units
- deploy
- view the all units list
- there should only be a Remove button
- start the simulator (ctrl+s)
- wait for some units to be in the error status
- navigate to the error units list
- there should be a 'Resolve & retry' button as well as a Remove button

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3433/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3441/
Test PASSed.

**Comment by hatched** [comment]
:+1:

**Comment by hatched** [comment]
QA :ok:

File: jujugui/static/gui/src/app/components/unit-list/test-unit-list.js [hunk 1]
```diff
@@ -264,17 +264,38 @@ describe('UnitList', () => {
...
-      action: output.props.children[2].props.buttons[0].action
+      action: buttonItems[0].action
     }];
     assert.deepEqual(output.props.children[2],
       <juju.components.ButtonRow
         buttons={buttons} />);
+    assert.equal(buttonItems.length, 1);
+  });
+
+  it('displays a Resolve & retry button for an error list', function() {
```
Comment (kadams54) [inline jujugui/static/gui/src/app/components/unit-list/test-unit-list.js]: Nitpick: inconsistent case.

**Comment by kadams54** [comment]
LGTM as well. :+1: and :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/1175

**Comment by kadams54** [comment]
Also a failure on the new `no-unused-vars` lint rule.

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3455/
Test PASSed.

**Pull request merged**